### PR TITLE
feat: support only webpack workflow

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "nativescript-cloud",
-  "version": "1.17.6",
+  "version": "1.17.7-2019-06-18-01",
```
Comment (KristianDD) [inline package.json]: 1.18.0

File: lib/services/cloud-platform-service.ts [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+	constructor(
+		private $injector: IInjector,
+		private $staticConfig: IStaticConfig
+	) {
+		const cliVersion = this.$staticConfig.version;
+		this.shouldUseOldPlatformService = semver.valid(cliVersion) && semver.lt(cliVersion, semver.prerelease(cliVersion) ? "5.4.0-2019-05-16-13277" : "6.0.0");
+	}
+
+	public get $nsCloudBuildCommandHelper(): IBuildCommandHelper {
+		return this.$injector.resolve("nsCloudBuildCommandHelper");
```
Comment (KristianDD) [inline lib/services/cloud-platform-service.ts]: Why we don't let the injector resolve this dependency?

File: lib/services/cloud-platform-service.ts [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+			const $platformService = this.$injector.resolve("platformService");
+			return $platformService.shouldInstall(device, projectData, config, config.outputPath);
+		}
+
+		const $deviceInstallAppService = this.$injector.resolve("deviceInstallAppService");
+		return $deviceInstallAppService.shouldInstall(device, config);
+	}
+
+	public async deployPlatform(platform: string, outputDirectoryPath: string): Promise<void> {
+		const $deployCommandHelper = this.$injector.resolve("deployCommandHelper");
```
Comment (KristianDD) [inline lib/services/cloud-platform-service.ts]: Why can't we leave the injector to resolve this in the constructor. We seem to use different method, but the same helper.

File: lib/services/cloud-platform-service.ts (hunk 2 above)
Comment (Fatme) [reply lib/services/cloud-platform-service.ts]: Because there is a cyclic dependency.

**Pull request merged**